crates/matrix-sdk/src/client/mod.rs

```diff
@@ -2147,7 +2174,7 @@ impl Client {
     ///     .await;
     /// })
     /// ```
-    #[instrument(skip(self, callback))]
+    #[instrument(skip_all, parent = &self.root_span)]
     pub async fn sync_with_callback<C>(
```

Comment:
This is a method that gets called from other sync methods. Using `parent =` here will make that relation invisible in logs, right?